### DISCUSSION TITLE
Why can’t my HERO12 Black - v2.10.70 firmware be updated successfully?

### DISCUSSION THREAD

**Discussion by joylc63** [gopro/labs#805] [General]
I have been unable to successfully update the HERO12 Black - v2.10.70 firmware using a TF card. I have tried many times but the same problem persists. I don’t know where the problem is. My gopro12 has been lightly modified and is currently version 2.00.70. firmware. I'm asking for help if anyone has the same problem as me.

**Comment by dnewman-gpsw** [comment]
Moved to discussion as this issue has been fixed in the latest Labs firmware, you just have to get that installed. I'm assuming your modifications involve a removed rear LCD. "Naked" or at least partly so. There was a change in the security on v2.00.70, designed to prevent naked configurations from bricking with the wrong firmware install, but this is now too restrictive.  Fortunately there is a workaround. Inside the UPDATE add a empty file called force.txt like this: 

![image](https://github.com/gopro/labs/assets/10856084/e7c99a3d-a511-4272-aafe-4e7bc89a295f)

The update to newer Labs on Naked HERO12s will now work.

**Reply by joylc63** [reply]
I have successfully updated the firmware according to your method, thank you very much!
